### PR TITLE
move users_emails.sto and orgs_glns.sto to /users and /orgs

### LINKED ISSUE #6002: Move users_emails.sto to users folders
users_emails.sto is really part of users data. So we should move it to users directory so that:
- it's in the same volume as users
- it's easier to zclone from prod this file also
- we don't need a separate mount for this very file.

### PR DESCRIPTION
**Pull request by stephanegigandet**
fixes #6002

### CODE REVIEW

File: cgi/reset_password.pl [hunk 1]
```diff
@@ -72,7 +72,7 @@
 	# Is it an email?
 
 		if ($id =~ /\@/) {
-			my $emails_ref = retrieve("$data_root/users_emails.sto");
+			my $emails_ref = retrieve("$data_root/users/users_emails.sto");
```
Comment (alexgarel) [inline cgi/reset_password.pl:75]: To ease transition on production, I propose to already move the files to users/ and orgs/ but symlink them in the upper directory.

**Review by alexgarel — APPROVED** [review]
:clap: cool one :-)

File: cgi/reset_password.pl (hunk 1 above)
Comment (alexgarel) [reply cgi/reset_password.pl:75]: pour l'info je l'ai fait en preprod, pour tester.

Comment (stephanegigandet) [reply cgi/reset_password.pl:75]: sounds good

**Pull request merged**